lib/hammer_cli_foreman/compute_profile.rb

```diff
@@ -0,0 +1,69 @@
+require 'hammer_cli_foreman/image'
+
+module HammerCLIForeman
+
+  class ComputeProfile < HammerCLIForeman::Command
+    resource :compute_profiles
+
+    class ListCommand < HammerCLIForeman::ListCommand
+
+      output do
+        field :id, _("Id")
+        field :name, _("Name")
+      end
+
+      build_options
+    end
+
+    class InfoCommand < HammerCLIForeman::InfoCommand
+
+      output ListCommand.output_definition do
+
+        HammerCLIForeman::References.taxonomies(self)
+        HammerCLIForeman::References.timestamps(self)
+
+        collection :compute_attributes, _("Compute attributes") do
+          field :id, _('Id')
+          field :name, _('Name')
+          field nil, _("Compute Resource"), Fields::SingleReference, :key => :compute_resource
+          field :vm_attrs, _("VM attributes")
```

Comment:
@shlomizadok see https://github.com/shlomizadok/hammer-cli-foreman/pull/1
The code will need more care. I'm sending it to share the idea rather than presenting a final solution.